From 4be60c90008ac48e72e819e078ce957fd003a509 Mon Sep 17 00:00:00 2001
From: Peter Hutterer <peter.hutterer@who-t.net>
Date: Fri, 12 Aug 2011 04:20:21 +0000
Subject: list: don't use defines for checking server version.

Otherwise we run into the old problem again: recompiling xinput against
newer inputproto headers will appear to change the version support,
potentially causing errors or other misbehaviours.

Signed-off-by: Peter Hutterer <peter.hutterer@who-t.net>
Reviewed-by: Jeremy Huddleston <jeremyhu@apple.com>
---
diff --git a/src/list.c b/src/list.c
index b4649eb..b791e81 100644
--- a/src/list.c
+++ b/src/list.c
@@ -248,14 +248,14 @@ static int
 list_xi2(Display *display,
          enum print_format format)
 {
-    int major = XI_2_Major,
-        minor = XI_2_Minor;
+    int major = 2,
+        minor = 0;
     int ndevices;
     int i, j;
     XIDeviceInfo *info, *dev;
 
     if (XIQueryVersion(display, &major, &minor) != Success ||
-        (major * 1000 + minor) < (XI_2_Major * 1000 + XI_2_Minor))
+        (major * 1000 + minor) < 2000)
     {
         fprintf(stderr, "XI2 not supported.\n");
         return EXIT_FAILURE;
--
cgit v0.9.0.2-2-gbebe
